server/middleware/context.go

```diff
@@ -0,0 +1,3 @@
+package middleware
+
+type ContextKeyAuthenticatedAccountAddress struct{}
```

Comment:
If you're not familiar with using `struct{}` for context keys, please see https://velog.io/@kimmachinegun/Go-context.WithValue-%EC%95%88%EC%A0%84%ED%95%98%EA%B2%8C-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0